lib/livebook_web/live/session_live/render.ex

```diff
@@ -504,33 +504,54 @@ defmodule LivebookWeb.SessionLive.Render do
-          <button
-            class="grow flex items-center text-gray-500 hover:text-gray-900 text-left"
-            data-el-sections-list-item
-            data-section-id={section_item.id}
-          >
-            <span class="flex items-center space-x-1">
-              <span><%= section_item.name %></span>
-              <%!--
-              Note: the container has overflow-y auto, so we cannot set overflow-x visible,
-              consequently we show the tooltip wrapped to a fixed number of characters
-              --%>
-              <span
-                :if={section_item.parent}
-                {branching_tooltip_attrs(section_item.name, section_item.parent.name)}
-              >
-                <.remix_icon
-                  icon="git-branch-line"
-                  class="text-lg font-normal leading-none flip-horizontally"
-                />
+        <div :for={section_item <- @data_view.sections_items} class="flex flex-col">
+          <div class="flex justify-between items-center">
+            <button
+              class="grow flex items-center text-gray-500 hover:text-gray-900 text-left"
+              data-el-sections-list-item
+              data-section-id={section_item.id}
+            >
+              <span class="flex items-center space-x-1">
+                <.remix_icon icon="font-size" class="text-lg font-normal leading-none" />
+                <span><%= section_item.name %></span>
+                <%!--
+                Note: the container has overflow-y auto, so we cannot set overflow-x visible,
+                consequently we show the tooltip wrapped to a fixed number of characters
+                --%>
+                <span
+                  :if={section_item.parent}
+                  {branching_tooltip_attrs(section_item.name, section_item.parent.name)}
+                >
+                  <.remix_icon
+                    icon="git-branch-line"
+                    class="text-lg font-normal leading-none flip-horizontally"
+                  />
+                </span>
               </span>
-            </span>
-          </button>
-          <.section_status
-            status={elem(section_item.status, 0)}
-            cell_id={elem(section_item.status, 1)}
-          />
+            </button>
+
+            <.section_status
+              status={elem(section_item.status, 0)}
+              cell_id={elem(section_item.status, 1)}
+            />
+          </div>
+
+          <ul :if={section_item.definitions != []} class="mt-2 ml-5 list-none items-center">
+            <li :for={definition <- section_item.definitions}>
+              <span class="flex items-center text-sm gap-1">
+                <.remix_icon icon="braces-line" class="font-normal leading-none" />
+                <span
+                  class="cursor-pointer font-mono text-ellipsis tooltip top"
+                  data-el-sections-list-module-item
```

Comment:
Nitpick: let's not assume it's a module:

```suggestion
                  data-el-sections-list-definition-item
```